README.md

```diff
@@ -104,6 +104,54 @@ kubectl --kubeconfig=kubeconfig get machines
 kubectl --kubeconfig=kubeconfig get machines -o yaml
 ```
 
+### Debugging your cluster
+
```

Comment:
is this for deployer or after cluster is up?

Reply:
Ah, I can see how that is confusing. That is for the deployer.

Reply:
Is this better @jichenjc